charts/soveren-dar-sensor/values.yaml

```diff
@@ -112,6 +112,13 @@ crawler:
       checkinterval: "24h"
       postgres:
         enabled: false
+        elements:
+            # PostgreSQL instance name
+          - name: ""
+            # PostgreSQL connection string
+            connectionString: ""
+            # Name of default database to connect to. If value is empty and connection string contains dbname, dbname will be used as default value
```

Comment:
# Default database name to use if not specified in the connection string.
# If this value is empty, the dbname from the connection string will be used.